independent-projects/tools/common/src/main/java/io/quarkus/cli/commands/ListExtensions.java

```diff
@@ -33,6 +33,8 @@ public ListExtensions(final BuildFile buildFile) throws IOException {
 
     public void listExtensions(boolean all, String format, String search) throws IOException {
         final Map<String, Dependency> installed = findInstalled();
+        if(format == null)
+            format = "name";
```

Comment:
Could you add curly braces, please?

Reply:
Shouldn't it be "concise", though? Just looking at the switch/default.